packages/itwinui-css/src/side-navigation/side-navigation.scss

```diff
@@ -86,11 +86,11 @@ $iui-side-navigation-icon-margins: calc(1.5 * var(--iui-size-m));
     border-radius: 0;
     border-left: none;
     border-right: none;
-    overflow: hidden;
+    isolation: isolate;
```

Comment:
Why did you remove `overflow: hidden`?

Reply:
If `overflow: hidden` was still applied, the blue stripe would not show because technically the stripe is positioned -2px outside of the button on the right hand side.